release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -2342,7 +2372,7 @@ oc create -f $PROJ-app.yaml
 +
 For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=1812212[*BZ#1812212*].
 
-* In {product-title} 4.3 and 4.4, if the user has the console open in multiple tabs, some sidebar links in the *Developer perspective* do not directly link to the project, and there is an unexpected change in the selected project. This will be resolved in a future release. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1839101[*BZ#1839101*])
+* In {product-title} 4.8, if the user has the console open in multiple tabs, some sidebar links in the *Developer perspective* do not directly link to the project, and there is an unexpected change in the selected project. This will be resolved in a future release. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1839101[*BZ#1839101*])
```

Comment:
*Developer perspective*/*Developer* perspective
This will be resolved in a future release. - seems redundant imho, take it or leave it :)

Reply:
Done